src/components/KToaster/ToastManager.ts

```diff
@@ -27,13 +29,30 @@ export default class ToastManager {
       return
     }
 
-    this.toastersContainer = document.createElement('div')
-    this.toastersContainer.id = toasterContainerId
-    document.body.appendChild(this.toastersContainer)
+    if (options?.zIndex) {
+      this.zIndex = options.zIndex
+    }
+
+    this.setupToastersContainer()
+  }
+
+  private setupToastersContainer(): void {
```

Comment:
you'll likely want to check that `document` exists just to be sure in case this is init during SSR